### ISSUE TITLE
https://dubbo.incubator.apache.org/ is very slow

### ISSUE THREAD

**Issue by diecui1202** [apache/incubator-dubbo-website#25]
![image](https://user-images.githubusercontent.com/1152291/42269879-fa609bca-7fb1-11e8-95a4-fe4b9159dea0.png)

Can't bear. Please fix it asap. Thanks very much!

**Comment by ralf0131** [comment]
Yes, I have encountered the same issue. I think the problem here is it loaded all the pages, including the documentation/blog posts, when access for the first time.

We should support on demand load of such pages.

**Comment by ralf0131** [comment]
Fixed.

**Comment by lovepoem** [comment]
Still slow

**Comment by lovepoem** [comment]
![image](https://user-images.githubusercontent.com/827112/42305487-b8b4b6e6-805d-11e8-945c-803bdb505e1d.png)
http://dubbo.apache.org/#/docs/user/quick-start.md?lang=en-us

**Comment by ralf0131** [comment]
In the documentation page, by default it will load all the documentation. I think this cause the slow page load. I will reopen this issue.
